addon/mixins/select.js

```diff
@@ -52,43 +32,83 @@ export default Mixin.create({
     }
   },
 
+  label: computed('value', 'content', function () {
+    let checkValue = this.get('value')
+
+    if (typeof this.get('value') === 'string' || typeof this.get('value') === 'boolean' ) {
+      checkValue = this.findValueObject(this.get('value'));
```

Comment:
Having `let checkValue = this.get('value')`, you may as well read it rather than calling `this.get('value')` more. Not a big deal either way though.